### DISCUSSION TITLE
API Now on air...

### DISCUSSION THREAD

**Discussion by radiobellissima** [AzuraCast/AzuraCast#3983] [General]
I have to fill in this, can you tell me what to put in order to have artist and automatic title?

https://services.getmeradio.com/api/song_update/?token=0000000000000000&station_id=00000&artist=Madonna&title=Frozen

Thanks
Marcello

**Comment by radiobellissima** [comment]
@BusterNeece can you help me on this? checking the get me radio service, I discovered that I still hadn't found a way to automatically update the title and artist.

**Reply by BusterNeece** [reply]
There are a lot of ways you could handle this, but as of right now they'd all require some custom code on your part to interface between the GetMeRadio API and AzuraCast's API format. If they can't fetch from our API, then you'll have to build something that takes our webhook in and then posts the update over to them.

You could also submit a feature request for us to implement it directly as a webhook. If there's enough support for it, we'd look into implementing it.

**Reply by radiobellissima** [reply]
Si is an aggregator used by many stations and offers many features. It would be enough to change that of tunein it works the same way there is an id and a key.

[How does the API work?](https://my.getmeradio.com/?rel=dashboard-faq#)
You can send us metadata to update your song information using our API in the format below. You have to let us know that you want to use the API instead of the stream metadata by selecting this in the 'Edit Station' section for the radio station (under 'My Stations' then click 'Edit' and scroll down to 'Stream Information' and change 'Metadata Source' from 'Streaming Server' to 'API' and click 'Update'.

Base URL : https://services.getmeradio.com/api/song_update/?

Then add the following fields:

token={your API token} Your token is visible in the 'My Stations' section, click 'Edit' and scroll down to the 'Stream Information' section. If the API Token field is blank, just click 'Update' at the bottom of the page and a token will be generated.
station_id={your station ID} Your station ID is visible in the 'My Stations' section and is a 3-5 digit number.
artist={artist name} The name of the artist playing.
title={song title} The title of the song playing.
An example would be as follows:

https://services.getmeradio.com/api/song_update/?token=0000000000000000&station_id=00000&artist=Madonna&title=Frozen

You can use the above to send us metadata for your radio station, please do not send more often than every 20 seconds or at song updates via your automation system.

**Comment by radiobellissima** [comment]
@BusterNeece Thanks.
